### ISSUE TITLE
Large number of read-only file system errors slowing down program launch

### ISSUE THREAD

**Issue by keftie22** [flathub/com.katawa_shoujo.KatawaShoujo#15]
When I launch the program I get many errors such as the snippet shown below:

```
Traceback (most recent call last):
  File "/app/share/renpy/renpy/script.py", line 650, in load_file
    with open(rpycfn, "wb") as f:
OSError: [Errno 30] Read-only file system: '/app/game/script-a2-hanako_JP.rpyc'
Traceback (most recent call last):
  File "/app/share/renpy/renpy/script.py", line 650, in load_file
    with open(rpycfn, "wb") as f:
OSError: [Errno 30] Read-only file system: '/app/game/script-a2-lilly.rpyc'
...
```

Full output log:

[output.txt](https://github.com/flathub/com.katawa_shoujo.KatawaShoujo/files/10430231/output.txt)

This seems to slow down the launch of the program quite a bit and it will sit on the splash screen for a long time waiting for these errors to cycle compared to before the renpy 8 upgrade and compared to non-flatpak version of this Renpy 8 port. 

| Version  | Launch time |
| ------------- | ------------- |
| non-flatpak   | 9 seconds  |
| x86_64 flatpak on optiplex 9010 with ssd  | 61 seconds  |
| aarch64 flatpak on oneplus 6 postmarketOS  | 85 seconds  |

Once it does launch everything else seems to be working great from this port other than this one small issue. I almost thought it wasn't going to work when it was taking this long to launch so I thought to check the terminal output and saw this. Hard drive users probably would be looking at 2 minutes or more if I were to guess.

I don't see any effect on the game itself so far but I've only clicked through the intro real quick just to test. It does worry me a bit that it might do something given that I do see some character names in the names of the files that it is throwing read-only errors on, but I really don't know enough to say if it will because I have no idea what it is trying to write to them or what their purpose is.

**Comment by Steinhagen** [comment]
This is fixed by pull request #16. The reason why this was incredibly slow was because the game didn't had the correct compiled binaries deployed. 

When starting the game, renpy was identifying that the binaries were not correct and was trying to regenerate them. This however is not good in flatpak since it mounts the /app directory as read-only, reason why we would get all those errors. The main purpose of those binaries are to speed up the game launch and execution.

If you can, please give the PR a try and check if it fixes your issues.

**Comment by keftie22** [comment]
Looks to be working great, only takes a few seconds to launch. Great work!

**Comment by FakeShemp** [comment]
Solved with #16
